src/components/Global/RadioButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import styles from './RadioButton.module.scss';
+
+type RadioButtonProps = {
+  id: string /*radioEasy*/;
+  name: string /*filter*/;
+  difficulty: string /*Easy>> */;
+  checked: boolean;
+  setChecked: (wasChecked: any) => void;
```

Comment:
change wasChecked type to boolean

Reply:
also setChecked is a function from useState if you use it like above, it means that the type of this function is not `() => void`, it's `React.Dispatch<React.SetStateAction<boolean>>`

Reply:
This function might have both cases, you can handle this by `React.Dispatch<React.SetStateAction<string>> | (wasChecked: boolean) => void;`